server/controllers/api/claim/publish/createPublishParams.js

```diff
@@ -11,19 +22,24 @@ const createPublishParams = (filePath, name, title, description, license, nsfw,
   }
   // provide default for license
   if (license === null || license.trim() === '') {
-    license = '';  // default to empty string
+    license = ''; // default to empty string
+  }
+  // provide default for licenseUrl
+  if (licenseUrl === null || licenseUrl.trim() === '') {
+    licenseUrl = ''; // default to empty string
```

Comment:
probably don't need the `//  default to empty string` comments, can clean up.

I would suggest using something like:
```
licenseUrl = licenseUrl === null ? '' : licenseUrl.trim();
```